rpm/ocrdesktop/ocrdesktop.spec

```diff
@@ -0,0 +1,30 @@
+Name:     ocrdesktop
+Version:  4.0
+Release:  2
+Summary:  Accessibility tool that converts content of the current window into a text
+License:  Public Domain
+
+Requires: python3 tesseract tesseract-langpack-eng python3-pillow python3-pyatspi python-pytesseract libwnck3 gtk3
+
+SOURCE0:  ocrdesktop
+SOURCE1: 09-keybindings-ocrdesktop
+
+%description
+RPM package for ocrdesktop
+
+%install
```

Comment:
Please see this file:
https://github.com/vojtapolasek/vojtux/blob/master/specs/vojtux-docs-en.spec
I believe the %install section can be omitted and the automatic macro can be used in %prep section.

Reply:
Added some changes, please, let me know if that's what you had in mind.